### ISSUE TITLE
Doesnt work.

### ISSUE THREAD

**Issue by GhostFire90** [mfosse/JoyCon-Driver#119]
Doesnt work, at all. I have my joycons paired. I have my bluetooth on and all i get is
![jc3](https://user-images.githubusercontent.com/31078396/52600448-822f4080-2e10-11e9-8724-327794e19b07.JPG) `` and no lights are set, no output, no vibrate. Nothing.

**Comment by Deecellar** [comment]
Did you connect

**Comment by GhostFire90** [comment]
Yeah, it's not the driver, it's the bluetooth apparently. I cant get the joycons to be "Connected" I've asked too many people and no one knows. If anyone has any ideas message me on discord Ghostfire04#0134

**Comment by li-darren** [comment]
It was working before, but now it is not. Same issue.

**Comment by Aidoboy** [comment]
Did you pair before running the program?

**Comment by Roklethe** [comment]
What I often have to do is removing both joycons from the list of bluetooth devices and then manually adding them both again to the list (it shows 'connected' then). But after I've done that joycon-driver.exe runs just fine.

**Comment by Feudspato** [comment]
Try not to close the window and use the vJoy Monitor. Mine wasn't working until I realized that the problem was that I was closing the cmd window of the joycon-driver.exe
